lib/spack/spack/package.py

```diff
@@ -1828,6 +1836,13 @@ def build_process():
             # check the filesystem for it.
             self.stage.created = False
 
+            # drop into the build environment of the package?
+            if self.drop_in_shell is not None:
+                from spack.main import SpackCommand
+                build_env = SpackCommand('build_env')
+                # FIXME Error: multiple values are not allowed for variant "build_type"
```

Comment:
Is this FIXME still valid? If so, we need to address it, if not we need to remove it.

Reply:
Yes, that's still an issue when I use the new option in this PR :thinking: